src/tactic/ui/tools/ingest_wdg.py

```diff
@@ -501,6 +502,12 @@ def get_content_wdg(my):
             folder_div.add_style("font-style: italic")
             folder_div.add_style("margin-bottom: 10px")
 
+        # update_process
+        update_process = my.kwargs.get("update_process")
+        if update_process:
+            my.update_process = update_process
+        else:
+            my.update_process = None
```

Comment:
can that be simplfied to 1 line: 
my.update_process = my.kwargs.get("update_process") or None